payment.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Ensures optimal rendering on mobile devices -->
+    <title>Make Payment</title>
+  </head>
+
+  <body>
+    <!-- Include the PayPal JavaScript SDK; replace "test" with your own sandbox Business account app client ID -->
+    <script src="https://www.paypal.com/sdk/js?client-id=AR5RGg6eq2jEloMZYTYdXSZUFLqEze-0EpDN54UE3j7uY2PIjXcXgpyh3gHJaOXJwLKdYinCRlC8IUsd&currency=USD"></script>
+
+    <!-- Set up a container element for the button -->
+    <div id="paypal-button-container"></div>
+
+    <script>
+      paypal.Buttons({
+
+        // Sets up the transaction when a payment button is clicked
+        createOrder: function(data, actions) {
+          return actions.order.create({
+            purchase_units: [{
+              amount: {
+                value: '77.44' // Can reference variables or functions. Example: `value: document.getElementById('...').value`
```

Comment:
When looking at the API, are the parameters like value things we can get from the database so that we do not have to set them individually like in like 23 (77.44)?